src/ResourceManager/Insights/Commands.Insights/Autoscale/NewAzureRmAutoscaleRuleCommand.cs

```diff
@@ -94,12 +94,6 @@ public class NewAzureRmAutoscaleRuleCommand : AzureRMCmdlet
         public ScaleDirection ScaleActionDirection { get; set; }
 
         /// <summary>
-        /// Gets or sets the scale action type
-        /// </summary>
-        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The scale action type for the setting")]
-        public ScaleType ScaleActionScaleType { get; set; }
```

Comment:
That is another consequence of the new spec. We currently support only one type of auto scale action (ChangeCount). The spec team suggested to remove the parameter and just send the constant value through the cable only.
I just added the parameter back, made it optional and made default to the only value supported ChangeCount. I am not using the parameter at all, but I'll keep it for backwards compatibility.